### DISCUSSION TITLE
Video Ambiguous: Caravel Counter timer UART

### DISCUSSION THREAD

**Discussion by yui0303** [bol-edu/HLS-SOC-Discussions#134] [General]
在 [Caravel Counter timer UART](https://www.youtube.com/watch?v=-o87eNkqmPo&t=793s) 影片中有提到下圖的例子。
影片中講者提到一開始 uart tx 是傳送 31 = b00011111，所以uart tx的輸出應該是 11111000，但該圖的輸出似乎是 10001100，不太清楚有沒有理解錯誤，謝謝 !
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/111721739/78113e10-ab6c-486e-ab61-c532d0e18a88)

**Comment by HurryLin** [comment]
你好, 可能是我在影片裡沒有描述清楚, 範例裡UART Tx傳送的字元”1”, 是傳送它的ASCII碼為31h(並非是十進制31)->00110001b.
然而UART會從LSB開始傳送, 因此最終按照圖中順序傳出. 謝謝!
